cmake/LibMeasurementKit.cmake

```diff
@@ -0,0 +1,27 @@
+## libmeasurement_kit
+
+file(
+  GLOB
+  MK_LIBRARY_SOURCES
+  "${MEASUREMENTKIT_ROOT}/include/measurement_kit/*.hpp"
+  "${MEASUREMENTKIT_ROOT}/include/measurement_kit/*/*.hpp"
+  "${MEASUREMENTKIT_ROOT}/src/libmeasurement_kit/*/*.c"
+  "${MEASUREMENTKIT_ROOT}/src/libmeasurement_kit/*/*.cpp"
```

Comment:
Here I use the `MEASUREMENTKIT_ROOT` variable in front. Since by default it's set to `CMAKE_SOURCE_DIR` it should not break anything when used from the root of the repo (`CMAKE_SOURCE_DIR` will always point to the absolute path of where the `CMakeLists.txt` is stored)